packages/paste-website/src/pages/components/spinner/index.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: Spinner
+package: '@twilio-paste/spinner'
+description: An icon component that can used in various loading situations.
+slug: /components/spinner/
+---
+
+import {graphql} from 'gatsby';
+import {Stack} from '@twilio-paste/stack';
+import {Spinner} from '@twilio-paste/spinner';
+import Changelog from '@twilio-paste/spinner/CHANGELOG.md';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {SiteLink} from '../../../components/SiteLink';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/spinner"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/spinner/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/spinner"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-spinner--default"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+### About Spinner
+
+The spinner is an icon component that can be used in situations where you would like to communicate a loading state to the user. It inherits a lot of properties from the [Icon component](/icon-system/).
+
+<Callout>
+  <CalloutTitle>Accessible animation</CalloutTitle>
+  <CalloutText>
+    The spinner animation will automatically respect user preferences set at the operating system level via the prefers
+    reduced motion API.
+  </CalloutText>
+</Callout>
+
+## Examples
+
+### Default
+
+The default Spinner settings are using the default icon size and color, which is designed to match our default text size and color.
+
+The only requirement is that you set whether it is a decorative Spinner or not, which follows the [same guidelines as icons](/icon-system/#accessibility).
+
+<Callout>
+  <CalloutTitle>Accessibility</CalloutTitle>
+  <CalloutText>
+    Please pay careful attention to our{' '}
+    <SiteLink to="/icon-system/#accessibility">icon accessibility guidelines</SiteLink>.
+  </CalloutText>
+</Callout>
```

Comment:
Do we need this, or can you put the specific accessibility guidelines we're asking people to pay attention to? Doesn't seem super helpful to me, and might almost repeat the paragraph above it with less detail.